main/src/androidTest/java/cgeo/geocaching/activity/waypoint/AbstractAddWaypointActivityTest.java

```diff
@@ -1,12 +1,10 @@
 package cgeo.geocaching.activity.waypoint;
 
-import cgeo.geocaching.EditWaypointActivity_;
-
 public abstract class AbstractAddWaypointActivityTest extends AbstractWaypointActivityTest {
     @Override
     protected void setUp() throws Exception {
         super.setUp();
-        setActivityIntent(new EditWaypointActivity_.IntentBuilder_(getInstrumentation().getContext()).geocode(getCache().getGeocode()).get());
-        getActivity();
+//        setActivityIntent(new EditWaypointActivity.IntentBuilder_(getInstrumentation().getContext()).geocode(getCache().getGeocode()).get());
+//        getActivity();
```

Comment:
Remove or reactivate it.

Reply:
This class will be deleted eventually after #13798 is resolved and I can fix the GUI tests for good. I keep it only as reference for later test refactoring. I will comment the class accordingly, hope it is then fine to keep it for now.